encoding/protobuf/error_test.go

```diff
@@ -184,7 +185,9 @@ func TestPbErrorToYARPCError(t *testing.T) {
 			assert.Equal(t, tt.message, status.Message(), "unexpected grpc status message")
 			assert.Len(t, status.Details(), len(tt.details), "unexpected details length")
 			for i, detail := range tt.details {
-				assert.Equal(t, detail, status.Details()[i])
+				if !proto.Equal(detail, status.Details()[i].(proto.Message)) {
+					t.Errorf("non comparable messages")
+				}
```

Comment:
This should be `assert.True(t, proto.Equal(detail, status.Details()[i].(proto.Message)))`